src/cmds/scripts/pbs_init.d.in

```diff
@@ -269,188 +268,183 @@ check_hostname() {
-	 if ${PBS_EXEC}/sbin/pbs_sched
-	 then
-            echo "PBS sched"
-         else
-            ret_val=$?
-            echo "pbs_sched startup failed, exit ${ret_val} aborting." >&2
-            return 1
-         fi
-      fi
-    fi
+	if [ "${PBS_START_MOM}" -gt 0 ]; then
+		if check_prog "mom" ; then
+			echo "PBS mom already running."
+		else
+			if [ -f ${pbslibdir}/init.d/limits.pbs_mom ]; then
+				. ${pbslibdir}/init.d/limits.pbs_mom
+			fi
+			check_maxsys
+
+			if  ${PBS_EXEC}/sbin/pbs_mom
+			then
+				echo "PBS mom"
+			else
+				ret_val=$?
+				echo "pbs_mom startup failed, exit ${ret_val} aborting." >&2
+				return 1
+			fi
+		fi
+	fi
 
-    if [ "${PBS_START_SERVER}" -gt 0 ] ; then
-      if check_prog "server" || check_prog "oldserver" ; then
-	echo "PBS Server already running."
-      else
-         if [ -f ${pbslibdir}/init.d/limits.pbs_server ] ; then
-             . ${pbslibdir}/init.d/limits.pbs_server
-         fi
-	 if ${PBS_EXEC}/sbin/pbs_server ; then
-            echo "PBS server"
-         else
-            ret_val=$?
-	    if [ $ret_val -eq 4 ] ; then
-		echo "pbs_server failed to start, will retry once in 30 seconds" >&2
-		sleep 30
-	        if ${PBS_EXEC}/sbin/pbs_server ; then
-		    echo "PBS server"
+	if [ "${PBS_START_SCHED}" -gt 0 ]; then
+		if check_prog "sched" ; then
+			echo "PBS scheduler already running."
 		else
-		    ret_val=$?
-            	    echo "pbs_server startup failed, exit ${ret_val} aborting." >&2
-		    return 1
+			if [ -f ${pbslibdir}/init.d/limits.pbs_sched ]; then
+				. ${pbslibdir}/init.d/limits.pbs_sched
+			fi
+			if ${PBS_EXEC}/sbin/pbs_sched
+			then
+				echo "PBS sched"
+			else
+				ret_val=$?
+				echo "pbs_sched startup failed, exit ${ret_val} aborting." >&2
+				return 1
+			fi
 		fi
-	    else
-		echo "pbs_server startup failed, exit ${ret_val} aborting." >&2
-		return 1
-	    fi
-	 fi
+	fi
 
-         if [ -r "${PBS_HOME}/server_priv/qmgr_startup" ]; then
-           echo "pbs_server evaluating ${PBS_HOME}/server_priv/qmgr_startup"
-           ${PBS_EXEC}/bin/qmgr <"${PBS_HOME}/server_priv/qmgr_startup"
-         fi
-      fi
-    fi
+	if [ "${PBS_START_SERVER}" -gt 0 ] ; then
+		if check_prog "server" || check_prog "oldserver" ; then
+			echo "PBS Server already running."
+		else
+			if [ -f ${pbslibdir}/init.d/limits.pbs_server ] ; then
+				. ${pbslibdir}/init.d/limits.pbs_server
+			fi
+			if ${PBS_EXEC}/sbin/pbs_server ; then
+				echo "PBS server"
+			else
+				ret_val=$?
+				if [ $ret_val -eq 4 ] ; then
+					echo "pbs_server failed to start, will retry once in 30 seconds" >&2
+					sleep 30
+					if ${PBS_EXEC}/sbin/pbs_server ; then
+						echo "PBS server"
+					else
+						ret_val=$?
+						echo "pbs_server startup failed, exit ${ret_val} aborting." >&2
+						return 1
+					fi
+				else
+					echo "pbs_server startup failed, exit ${ret_val} aborting." >&2
+					return 1
+				fi
+			fi
+
+			if [ -r "${PBS_HOME}/server_priv/qmgr_startup" ]; then
```

Comment:
I think we can get rid of this, but can you please check once? last time I looked for this I couldn't find anything in guides.